tests/test_widget.py

```diff
@@ -1,16 +1,22 @@
+from contextlib import nullcontext as does_not_raise
+from decimal import Decimal
+
 import pytest
 
 from textual.css.errors import StyleValueError
+from textual.css.scalar import Scalar, Unit
 from textual.widget import Widget
 
 
 @pytest.mark.parametrize(
-    "set_val, get_val, style_str", [
+    "set_val, get_val, style_str",
```

Comment:
Black formatting was not applied on this file.. It's the case now that I edited it :slightly_smiling_face: